Gemfile

```diff
@@ -138,6 +138,9 @@ group :test, :development do
   gem 'pry-rails'
   gem 'action-cable-testing'
   gem "webmock", "~> 3.14"
+  gem 'rubocop-rails', require: false
+  gem 'rubocop-performance', require: false
+  gem 'rubocop-rspec', require: false
```

Comment:
This looks really good @kunmi02. I was able to run rubocop in my local env. Why do we need these additional gems? Also, I don't think we need the `rubocop-rspec` gem since we don't use RSpec as our test suite.

Reply:
Most of the outdated commands on the rubocop file are now wrapped in the gems I added.  For the rubocop-rspec, I have removed it from the gemfile.